src/neuroconv/nwbconverter.py

```diff
@@ -53,16 +52,15 @@ def get_conversion_options_schema(cls):
             description="Schema for the conversion options",
             version="0.1.0",
         )
-        for interface_name, data_interface in cls.data_interface_classes.items():
+        for interface_name, data_interface in self.data_interface_objects.items():
             conversion_options_schema["properties"].update(
                 {interface_name: unroot_schema(data_interface.get_conversion_options_schema())}
             )
         return conversion_options_schema
 
-    @classmethod
-    def validate_source(cls, source_data: Dict[str, dict], verbose: bool = True):
+    def validate_source(self, source_data: Dict[str, dict], verbose: bool = True):
```

Comment:
Shouldn't the source schema (and its validation) still be class methods for the `NWBConverter`?

The only issue related to #292 is specific to the conversion options